hw1/libs/patch.js

```diff
@@ -0,0 +1,15 @@
+const { COLOR } = require('./env');
+const chalk = require('chalk');
+const log = console.log;
+
+global.console.log = (...args) => {
+    log(chalk[COLOR]("==="), chalk[COLOR](...args), chalk[COLOR]("==="))
```

Comment:
```suggestion
    log(chalk[COLOR]("===", ...args, "==="));
```
Но вообще, `"==="` - лишнее, это для примера на лекции показывал.